client/src/pages/ExecutePayment.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+
+import { Loader } from 'semantic-ui-react';
+import { useLocation, useHistory } from 'react-router-dom';
+
+import usePayment from '../hooks/payment';
+
+import { ExecutePaymentRequest } from '../proto/payment_pb';
+
+const ExecutePayment = props => {
+    const { state, methods } = usePayment();
+
+    const location = useLocation();
+    const history = useHistory();
+
+    const { executePayment } = methods;
+    useEffect(() => {
+        const searchParams = new URLSearchParams(location.search);
+        const paymentId = searchParams.get('paymentId');
+        const payerId = searchParams.get('PayerID');
+        console.log(paymentId, payerId);
+        const request = new ExecutePaymentRequest();
+        request.setPaymentid(paymentId);
```

Comment:
why is not camel case

Reply:
This is auto-generated source from grpc. The problem is in the library

Reply:
all clear now